content/pt/docs/concepts/signals/logs.md

```diff
@@ -0,0 +1,255 @@
+
+{{% signal-support-table "logs" %}}
+
+## Logs estruturados, não estruturados e semiestruturados {#structured-unstructured-and-semistructured-logs}
+
+Tecnicamente o OpenTelemetry não distingue entre logs estruturados e não
+estruturados. Você pode usar qualquer log que tiver com o OpenTelemetry. No
+entanto, nem todos os formatos de log são igualmente úteis! Logs estruturados,
+em particular, são recomendados para observabilidade em produção porque são
+fáceis de analisar e interpretar em escala. A seção a seguir explica as
+diferenças entre logs estruturados, não estruturados e semiestruturados.
+
+### Logs estruturados {#structured-logs}
+
+Um log estruturado é aquele que segue um formato consistente e legível por
+máquina. Para aplicações, um dos formatos mais comuns é o JSON:
+
+```json
+{
+  "timestamp": "2024-08-04T12:34:56.789Z",
+  "level": "INFO",
+  "service": "user-authentication",
+  "environment": "production",
+  "message": "User login successful",
+  "context": {
+    "userId": "12345",
+    "username": "johndoe",
+    "ipAddress": "192.168.1.1",
+    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36"
+  },
+  "transactionId": "abcd-efgh-ijkl-mnop",
+  "duration": 200,
+  "request": {
+    "method": "POST",
+    "url": "/api/v1/login",
+    "headers": {
+      "Content-Type": "application/json",
+      "Accept": "application/json"
+    },
+    "body": {
+      "username": "johndoe",
+      "password": "******"
+    }
+  },
+  "response": {
+    "statusCode": 200,
+    "body": {
+      "success": true,
+      "token": "jwt-token-here"
+    }
+  }
+}
+```
+
+e para componentes de infraestrutura, o _Common Log Format_ (CLF) é
+frequentemente usado:
+
+```text
+127.0.0.1 - johndoe [04/Aug/2024:12:34:56 -0400] "POST /api/v1/login HTTP/1.1" 200 1234
+```
+
+Também é comum ter logs estruturados em diferentes formatos juntos. Por exemplo,
+um log no formato _Extended Log Format_ (ELF) pode combinar JSON com os dados
+separados por espaços em um log CLF.
+
+```text
+192.168.1.1 - johndoe [04/Aug/2024:12:34:56 -0400] "POST /api/v1/login HTTP/1.1" 200 1234 "http://example.com" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36" {"transactionId": "abcd-efgh-ijkl-mnop", "responseTime": 150, "requestBody": {"username": "johndoe"}, "responseHeaders": {"Content-Type": "application/json"}}
+```
+
+Para aproveitar ao máximo este log, analise tanto os dados em formato JSON
+quanto os em formato ELF, utilizando um formato compartilhado para facilitar a
+análise em um backend de observabilidade. O `filelogreceiver` no
+[OpenTelemetry Collector](/docs/collector) contém maneiras padronizadas de
+analisar logs como estes.
+
+Logs estruturados são a melhor forma de usar logs. Por serem emitidos em um
+formato consistente, os logs são analisados diretamente, o que facilita o
```

Comment:
```suggestion
formato consistente, eles são simples de extrair informações, o que facilita o
```

"straightforward" pode ser usado como "direto", mas é mais quando usado para descrever a forma de fazer algo ("he was very straightforward in his speech", "ele foi muito direto no seu discurso"). Nesse caso é mais no sentido ser algo simples ("this is a straightforward task").